### ISSUE TITLE
Snapshots fail to get saved after powering OFF/Powering ON an etcd node

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#16551]
Environment information: v2.1.2-rc9

Steps to reproduce (least amount of steps as possible):

1. Create a 3 node DO cluster(RKE) with snapshots enabled every 5m
N1 , N2, N3 with all 3 roles in each of them
2. Ensure snapshots are created in /opt/rke/etcd-snapshots directory. etcd-rolling-snapshots should be running 
3. Power OFF N1 
4. Power ON N1

etcd-rolling-snapshots fails to start on N1. Snapshots are not saved after Power OFF and Power ON.
```
root@soumyasnaptest1:/opt/rke/etcd-snapshots# sudo docker ps -a|grep etcd
a65da9a8a2f2        rancher/rke-tools:v0.1.15                                                                                  "/opt/rke-tools/rk..."   About an hour ago   Exited (2) 30 minutes ago                         etcd-rolling-snapshots
c07c96cae5d0        rancher/coreos-etcd:v3.2.18                                                                                "/usr/local/bin/et..."   About an hour ago   Up 17 minutes                                     etcd



```

**Comment by alena1108** [comment]
Fix is available in rancher/rancher:2.1.2-rc11

**Comment by sangeethah** [comment]
Tested with rancher server version: v2.1.2-rc11

Steps to reproduce :

Create a 4 node ( 3 etcd and 1 node with control and worker role)  DO cluster(RKE) with snapshots enabled every 5m
Ensure snapshots are created in /opt/rke/etcd-snapshots directory. etcd-rolling-snapshots should be running.
Power OFF one of the etcd node.
Power ON the etcd node after while.
etcd-rolling-snapshots starts on the etcd node as expected.
New snapshots gets created at the interval set as expected.
